### PR TITLE
Home Page Responsiveness Fixed

### PR DESCRIPTION
**Pull request by Lakshay-Pruthi**
This PR fixes the following issues:
#96 #97 

## Changes proposed
The homepage has been made responsive now for all kind of media screens.
Also, the misalignment in the navbar contents and founder sections has been fixed.
Some small changes are also made which can be understood by viewing the screenshots attached below.

- [x] My code follows the code style of this project.
- [x] The title of my pull request is a short description of the requested changes.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Screenshots -->
1.
Before

After

2.
Before

After

3.Before

After

4.
Before

After
Founder image and text aligned

5.
Before

After
YouTube button aligned in the center and with the nav items

<a href="https://gitpod.io/#https://github.com/commclassroom/commclassroom/pull/99"></a>

### CODE REVIEW

**Comment by Lakshay-Pruthi** [comment]
Note: Some files may seem a lot of code has been deleted and added but this is not the case. As, Github also marks removing of spaces as changes (Code refactoring).
Some of the files where minor changes are done includes -->
1.About.css
2.Courses.css
3.Events.css
4.Mentorship.css
5.Podcast.css

File: src/styles/About.css [hunk 1]
```diff
@@ -1,99 +1,100 @@
...
-      letter-spacing:2px;
-      word-spacing: 12px;
-    }
-
-    .about_img{ 
-      width: 320px;
-      height: 250px;
-      margin-left: 15px;
-    }
-  }
```
Comment (SuperAayush) [inline src/styles/About.css:99]: Why is all the code of About.css removed and classes of blogs section is added here?

**Review by SuperAayush — CHANGES_REQUESTED** [review]

**Comment by AnuragThePathak** [comment]
You need to resolve conflicts to get it merged.

**Comment by Lakshay-Pruthi** [comment]
@AnuragThePathak Already resolved in previous commit. It is still showing conflicts because of the the last link added to the list items which is good to go. No problem there.

**Comment by Lakshay-Pruthi** [comment]
@AnuragThePathak It is showing conflict because of the highlighted text
image
I can remove it if you want but it will create issues in Navbar responsiveness

**Comment by Lakshay-Pruthi** [comment]
@AnuragThePathak Resolved!! Didn't knew about marking it as resolved.

**Review by ManishBisht777 — COMMENTED** [review]
I think its good to have one issue solved at a time , so it will be easy for others to review it

**Comment by Lakshay-Pruthi** [comment]
> I think its good to have one issue solved at a time , so it will be easy for others to review it

Check this https://github.com/WeMakeDevs/wemakedevs/issues/96#issuecomment-1296941457

**Pull request closed without merging**